internal/service/bedrockagent/agent.go

```diff
@@ -366,6 +365,12 @@ func (r *agentResource) Delete(ctx context.Context, request resource.DeleteReque
 	}
 }
 
+func (r *agentResource) ImportState(ctx context.Context, req resource.ImportStateRequest, resp *resource.ImportStateResponse) {
```

Comment:
Can now remove embedded `framework.WithImportByID`.

Reply:
Nice catch! I removed this initially and think I left that change on a different branch.

Reply:
Fixed.